### ISSUE TITLE
Application crashes when trying to disable background updates

### ISSUE THREAD

**Issue by poqueque** [transistorsoft/flutter_background_geolocation#969]
## Your Environment
* Plugin version: 4.9.0
* Platform: Android
* OS version: Android 12
* Device manufacturer / model: Samsung Galaxy S22
* Flutter info (`flutter doctor`):
[✓] Flutter (Channel stable, 3.7.5, on macOS 12.6 21G115 darwin-arm64, locale es-ES)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc1)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
[✓] Android Studio (version 2022.1)
[✓] IntelliJ IDEA Community Edition (version 2022.2.4)
[✓] VS Code (version 1.75.0)
[✓] Connected device (4 available)
[✓] HTTP Host Availability

* Plugin config:
```dart <-- Syntax highlighting: DO NOT REMOVE -->

  static bg.Config stopConfig = bg.Config(
    //disableLocationAuthorizationAlert: true,
    locationAuthorizationRequest: "Always",
    debug: false,
    //Setting true makes a sound on each location update
    desiredAccuracy: bg.Config.DESIRED_ACCURACY_NAVIGATION,
    isMoving: true,
    distanceFilter: 100,
    locationUpdateInterval: 1000 * 5,
    fastestLocationUpdateInterval: 1000 * 5,
    stopOnTerminate: true,
    startOnBoot: false,
    preventSuspend: false,
    enableHeadless: false,
    locationAuthorizationAlert: {
      'titleWhenNotEnabled': 'Set Location to "Always"',
      'titleWhenOff': 'Set Location to "Always"',
      'instructions':
          '{applicationName} needs to access your location data so it can provide accurate geolocation to you, your team, and first responders during urgent or emergency situations.',
      'cancelButton': 'Cancel',
      'settingsButton': 'Update',
    },
    backgroundPermissionRationale: bg.PermissionRationale(
        title:
            "Allow {applicationName} to access this device's location even when the app is closed or not in use.",
        message:
            "{applicationName} needs to access your location data so it can provide accurate geolocation to you, your team, and first responders during urgent or emergency situations.",
        positiveAction: 'Change to "{backgroundPermissionOptionLabel}"',
        negativeAction: 'Cancel'),
  );
```

## Expected Behavior
Application continues working

## Actual Behavior
Application crashes

## Steps to Reproduce
It started to happen after upgrading all the application to last Flutter, Gradle and Android versions. Its working properly on iOS.

## Context
The logs show a FATAL Exception on com.transistorsoft.locationmanager.location.TSLocationManager.removeLocationUpdates

## Debug logs

<details>
	<summary>Logs</summary>

``` <!-- syntax-highligting:  DO NOT REMOVE -->
2023-02-28 23:07:50.126 21583-21669 AndroidRuntime          com.northshoreresearch.clarityalarm  E  FATAL EXCEPTION: pool-19-thread-3
                                                                                                    Process: com.XXX.XXX, PID: 21583
                                                                                                    java.lang.IncompatibleClassChangeError: Found interface com.google.android.gms.location.FusedLocationProviderClient, but class was expected (declaration of 'com.google.android.gms.location.FusedLocationProviderClient' appears in /data/app/~~JgyGkuIAmjIBoIuN4x7nuw==/com.northshoreresearch.clarityalarm-gpPFK56UxgOaONE73o-WCg==/base.apk)
                                                                                                    	at com.transistorsoft.locationmanager.location.TSLocationManager.removeLocationUpdates(Unknown Source:41)
                                                                                                    	at com.transistorsoft.locationmanager.location.TSLocationManager.stopUpdatingLocation(Unknown Source:0)
                                                                                                    	at com.transistorsoft.locationmanager.location.TSLocationManager.stop(Unknown Source:0)
                                                                                                    	at com.transistorsoft.locationmanager.service.TrackingService.stop(Unknown Source:100)
                                                                                                    	at com.transistorsoft.locationmanager.adapter.BackgroundGeolocation.c(Unknown Source:2)
                                                                                                    	at com.transistorsoft.locationmanager.adapter.BackgroundGeolocation.$r8$lambda$oSDeWXxQl1jmikdjYQwff60damw(Unknown Source:0)
                                                                                                    	at com.transistorsoft.locationmanager.adapter.BackgroundGeolocation$$ExternalSyntheticLambda0.run(Unknown Source:4)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
                                                                                                    	at java.lang.Thread.run(Thread.java:1012)
```

</details>

**Comment by christocracy** [comment]
Duplicate #933

**Comment by poqueque** [comment]
If I understand correctly #933 it says it should be fixed in version 4.9.0

**Comment by christocracy** [comment]
933 now allows you to configure `ext.playServicesLocationVersion >= 21.0.0"`

One of your other dependence's increased that version (likely with a hard-coded version), forcing you to increase the version background-geolocation uses.
